code/game/objects/items/food/salad.dm

```diff
@@ -0,0 +1,169 @@
+	eatverbs = list("devour","nibble","gnaw","gobble","chomp") //who the fuck gnaws and devours on a salad
+
+/obj/item/food/salad/aesirsalad
+	name = "\improper Aesir salad"
+	desc = "A salad consisting of the botanical experiment known as ambrosia vulgaris. Primarily made for its medicinal use, though it's rather bitter."
+	icon_state = "aesirsalad"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 8,
+		/datum/reagent/medicine/omnizine = 10,
+		/datum/reagent/consumable/nutriment/vitamin = 12
+	)
+	tastes = list("leaves" = 1)
+	foodtypes = VEGETABLES | FRUIT
+
+/obj/item/food/salad/herbsalad
+	name = "herb salad"
+	desc = "A tossed salad with apple slices atop it."
+	icon_state = "herbsalad"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 8,
+		/datum/reagent/consumable/nutriment/vitamin = 6
+	)
+	tastes = list("leaves" = 1, "apple" = 1)
+	foodtypes = VEGETABLES | FRUIT
+
+/obj/item/food/salad/validsalad
+	name = "full salad"
+	desc = "A salad consisting of a traditional caesar salad, with the inclusion of meatballs and fried potato chunks."
+	icon_state = "validsalad"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 5,
+		/datum/reagent/consumable/nutriment/protein = 5,
+		/datum/reagent/consumable/doctor_delight = 8,
+		/datum/reagent/consumable/nutriment/vitamin = 6
+	)
+	tastes = list("leaves" = 1, "potato" = 1, "meat" = 1, "valids" = 1)
+	foodtypes = VEGETABLES | MEAT | FRIED | FRUIT
+
+/obj/item/food/salad/fruit
+	name = "fruit salad"
+	desc = "A bowl of tossed fruits without their peels and skins."
+	icon_state = "fruitsalad"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 9,
+		/datum/reagent/consumable/nutriment/vitamin = 5
+	)
+	tastes = list("fruit" = 1)
+	foodtypes = FRUIT
+
+/obj/item/food/salad/jungle
+	name = "jungle salad"
+	desc = "A tropical-focused fruit salad, noticeably sweet and tart."
+	icon_state = "junglesalad"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 11,
+		/datum/reagent/consumable/banana = 5,
+		/datum/reagent/consumable/nutriment/vitamin = 7
+	)
+	tastes = list("fruit" = 1, "the jungle" = 1)
+	foodtypes = FRUIT
+
+/obj/item/food/salad/citrusdelight
+	name = "citrus delight"
+	desc = "A bowl full of various slices of citrus, resulting in an incredibly tart fruit salad."
+	icon_state = "citrusdelight"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 11,
+		/datum/reagent/consumable/nutriment/vitamin = 7
+	)
+	tastes = list("sourness" = 1, "leaves" = 1)
+	foodtypes = FRUIT
+
+/obj/item/food/salad/ricebowl
+	name = "ricebowl"
+	desc = "A bowl of loose, pearled rice grains."
+	icon_state = "ricebowl"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 4)
+	tastes = list("rice" = 1)
+	foodtypes = GRAIN | RAW
+	microwaved_type = /obj/item/food/salad/boiledrice
+
+/*
+/obj/item/food/salad/ricebowl/make_microwaveable()
+	AddElement(/datum/element/microwavable, /obj/item/food/salad/boiledrice)
+*/
```

Comment:
think you can ditch this too

Reply:
this is a surprise tool that will help us later